### ISSUE TITLE
Self-supervised pre-training checkpoint and finetuning configuration

### ISSUE THREAD

**Issue by orsveri** [google-deepmind/trecvit#2]
Hi! 
Thank you for sharing your code!

Could you also share the pre-trained checkpoints for self-supervised pre-training? 
Many works demonstrate the advantage of self-supervised pretraining over supervised classification, so I believe this checkpoint could provide a much better backbone for downstream applications! 

Also, what batch size did you use for finetuning?
